README.md

```diff
@@ -380,7 +380,46 @@ Staltz can see Arj has invited Mix because he's replicating Arj's
 
 ## 5. Security Considerations
 
-TBD
+### 5.1. Degraded confidentiality at scale
+
+As noted in the [private-groups-original-notes], SSB Private Groups are based on
+symmetric encryption and static keys (no key rotation), which means it cannot
+guarantee [perfect-forward-secrecy] neither [post-compromise-security]. This
+makes SSB Private Groups less likely to guarantee confidentiality at scale, when
+a group has dozens or hundreds of members. If any of the members leaks the group
+secret, all past communication in the group is compromised.
+
+This also means that any group member can add new members to the group. They may
+even do so without publishing a `group/add-member` message, which effectively
+allows any group member to invite a third party that will remain undetected by
+existing group members.
+
+SSB Private Groups with several members should thus not be considered
+confidential nor forward secret, and should be used with caution. Our security
+model assumes that the group is as trustworthy as the _least_ trustworthy member
+in the group.
+
+The choice of symmetric encryption (as opposed to, e.g. hash-ratchet or
+double-ratchet) was made because of efficiency. Symmetric encryption means
+adding new members or publishing new content is `O(1)` fast, and the system is
+overall simple to implement.
+
+### 5.2. Membership analysis attacks
+
+A group feed is discoverable only by the group members, because its announcement
+is encrypted. However, an eavesdropper who can replicate a peer's metafeed tree
+will notice the presence of these encrypted message on shard feeds. If the
+eavesdropper can do that for several peers, then they may perform an analysis
```

Comment:
maybe say: "metadata analysis" instead of just analysis here.

Reply:
done